neonvm/controllers/metrics.go

```diff
@@ -112,7 +141,11 @@ func WithMetrics(reconciler reconcile.Reconciler, rm ReconcilerMetrics, cntrlNam
 }
 
 func (d *wrappedReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	log := log.FromContext(ctx)
+
+	now := time.Now()
```

Comment:
brief thought - wonder if it'd be worthwhile logging the start of the reconcile operation? may be too verbose... something to consider, perhaps for later.

Reply:
Yeah, before we had 0 lines per reconciliation in logs, if no errors.  Let's have one for now.